compiler/damlc/daml-stdlib-src/DA/Internal/Interface.daml

```diff
@@ -3,6 +3,7 @@
 
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
probably don't need this if nothing else changed in this file?

Reply:
ah, it's likely because of the split from the larger PR. Please move the change to the sub PR that uses the extensions